files/enroll.sh

```diff
@@ -13,6 +13,7 @@ PX_MEMBER_ENROLLMENT_PASSWORD=${PX_MEMBER_ENROLLMENT_PASSWORD:-66705b9ce583ffb9c
 # include libaries
 libraries="helper_libs.sh xroad_libs.sh"
 for l in $libraries; do
+  # shellcheck source=libs/*.sh
```

Comment:
Let's do this in our CI, shellcheck seems useful.

Reply:
Yes, removed.